src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -0,0 +1,176 @@
+package osmesa.analytics.oneoffs
+
+import cats.data.{Validated, ValidatedNel}
+import cats.implicits._
+import com.monovore.decline.{CommandApp, Opts}
+import geotrellis.vector._
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions.{explode, from_unixtime, to_timestamp, udf}
+import org.locationtech.geomesa.spark.jts.{udf => _, _}
+import org.locationtech.jts.{geom => jts}
+import org.locationtech.jts.geom.prep._
+import osmesa.analytics.Analytics
+import vectorpipe.sources.Source
+
+import java.net.URI
+
+// These only required for test getAreaIndex implementation
+import geotrellis.vector.io._
+import geotrellis.vector.io.json._
+import spray.json._
+
+object StreamingAOIMonitor extends CommandApp(
+  name = "streaming-aoi-monitor",
+  header = "Streaming AOI Monitor",
+  main = {
+    val intervalOpt =
+      Opts
+        .option[String]("interval", help = "Period of time to aggregate over (d=daily, w=weekly, m=monthly)")
```

Comment:
There was a conversation in the doc a day or so ago with Ross about removing monthly for now in order to simplify the initial ask. Let's do that here so that the extra option doesn't propagate further into the work we're doing.